ignite/services/network/client.go

```diff
@@ -31,3 +35,59 @@ func (n Network) CreateClient(
 	}
 	return createClientRes.ClientID, nil
 }
+
+// verifiedClientIDs fetches the verified client ids from SPN by launch id
+func (n Network) verifiedClientIDs(ctx context.Context, launchID uint64) ([]string, error) {
+	res, err := n.monitoringConsumerQuery.
+		VerifiedClientIds(ctx,
+			&monitoringctypes.QueryGetVerifiedClientIdsRequest{
+				LaunchID: launchID,
+			},
+		)
+
+	if cosmoserror.Unwrap(err) == cosmoserror.ErrNotFound {
+		return nil, ErrObjectNotFound
+	} else if err != nil {
+		return nil, err
+	}
+	return res.ClientIds, nil
+}
+
+// RewardIBCInfo returns IBC info to relay packets for a chain to claim rewards.
+func (n Network) RewardIBCInfo(ctx context.Context, launchID uint64) (networktypes.RewardIBCInfo, error) {
+	clientStates, err := n.verifiedClientIDs(ctx, launchID)
+	if err != nil {
+		return networktypes.RewardIBCInfo{}, err
+	}
+	if len(clientStates) == 0 {
+		return networktypes.RewardIBCInfo{}, ErrObjectNotFound
+	}
+
+	clientID := clientStates[0]
+
+	connections, err := n.node.clientConnections(ctx, clientID)
+	if err != nil && err != ErrObjectNotFound {
+		return networktypes.RewardIBCInfo{}, err
+	}
+	if err == ErrObjectNotFound || len(connections) == 0 {
```

Comment:
I'm not comfortable with direct error comparisons like that, this is often a source of nasty bugs. Typically if tomorrow someone adds a change and wraps the error, this will reverse the condition. 

`errors.Is(err, ErrObjectNotFound)` is more reliable.

Reply:
[91c9bcf](https://github.com/ignite/cli/pull/2516/commits/91c9bcf8edcdba8eaad92ed15fbefc61c5177485)